packages/core/src/queries/resources.ts

```diff
@@ -0,0 +1,70 @@
+import { ResourceServerDBEntry, ResourceServers } from '@logto/schemas';
+import { sql } from 'slonik';
+
+import { buildInsertInto } from '@/database/insert-into';
+import pool from '@/database/pool';
+import { buildUpdateWhere } from '@/database/update-where';
+import { convertToIdentifiers, OmitAutoSetFields } from '@/database/utils';
+import RequestError from '@/errors/RequestError';
+
+const { table, fields } = convertToIdentifiers(ResourceServers);
+
+export const findAllResources = async () =>
```

Comment:
`ResourceServers` would be better?
`resource` is tend to represent a specific entity of data.
something like "resource servers provide resources for user`

Reply:
ditto